### PR TITLE
deleted last phrase from README

### PR DESCRIPTION
**Pull request by marixko**
README is now less aggressive to the reader

### CODE REVIEW

**Review by camipacifici — APPROVED** [review]
Looks good to me!

File: participants/Nakazono/doc/README.md [hunk 1]
```diff
@@ -10,4 +10,3 @@ and this other part does something.  I forget why that I did it:
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
```
Comment (AG-Caneppa) [inline participants/Nakazono/doc/README.md:13]: Now the README is not aggressive ENOUGH! >:(

**Review by AG-Caneppa — COMMENTED** [review]
Sorry I'm just testing this feature

File: participants/Nakazono/doc/README.md [hunk 2]
```diff
@@ -10,4 +10,3 @@ and this other part does something.  I forget why that I did it:
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
```
Comment (fatma2585) [inline participants/Nakazono/doc/README.md:12]: That's Optimistic

**Review by fatma2585 — APPROVED** [review]

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**